diff --git a/node_modules/ngx-quill/lib/quill-editor.component.d.ts b/node_modules/ngx-quill/lib/quill-editor.component.d.ts
index ebe15b3..393905c 100644
--- a/node_modules/ngx-quill/lib/quill-editor.component.d.ts
+++ b/node_modules/ngx-quill/lib/quill-editor.component.d.ts
@@ -1,6 +1,7 @@
 import { DomSanitizer } from '@angular/platform-browser';
 import { QuillModules, CustomOption, CustomModule } from './quill-editor.interfaces';
-import QuillType, { Delta } from 'quill';
+import QuillType from 'quill';
+import Delta from 'quill';
 import { AfterViewInit, ChangeDetectorRef, ElementRef, EventEmitter, Injector, NgZone, OnChanges, OnDestroy, Renderer2, SimpleChanges } from '@angular/core';
 import { ControlValueAccessor, Validator } from '@angular/forms';
 import { QuillService } from './quill.service';
